assets/js/nav-accordion.js

```diff
@@ -9,5 +9,8 @@ $(document).ready(function() {
       $(this).attr('aria-expanded','false');
     });
   });
-
+  var currentNav = $('#nav').find('.current');
+  var navHeight = $('#nav').height();
+  var itemHeight = currentNav.height();
+  $('#nav').scrollTop(currentNav.position().top-(navHeight/2)+itemHeight);
```

Comment:
Why the additional JavaScript?

Reply:
Centers the selected nav item so that the user's currently selected nav item stays in view.